test/integration/config/esm-good/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
see my above comments around eslint configs and file extenions.  you should be able to do that and remove any overrides and `// eslint-disable`

Reply:
I disabled eslint in a less aggressive way.

Reply:
Wait, unless you mean that I shouldn't ignore `newrelic.cjs`?